public/nextjs_migration/client/js/dialog.js

```diff
@@ -31,7 +31,7 @@ function handleEvent (e) {
       openDialog(e.target.href)
       break
     case e.target.matches('button[data-dialog]'):
-      openDialog(`dialog/${e.target.dataset.dialog}`)
+      openDialog(`/api/v1/dialog/${e.target.dataset.dialog}`)
```

Comment:
Can we keep it at the present location? I think back-end will want to scope `/api/v1/` to APIs they have clear API contracts on, so we might be getting in their way if we pollute their namespace. (Also, it's not an API in the commonly-understood sense (i.e. returning JSON) anyway.)

Reply:
I don't think it'll be in the way, it is a little odd as an API though I agree (there are some HTML-over-the-wire frameworks out there fwiw)

Reply:
> Can we keep it at the present location? […]

See https://github.com/mozilla/blurts-server/pull/3142#discussion_r1235999490.